packages/lbfgs/lbfgs.0.9.5/opam

```diff
@@ -0,0 +1,39 @@
+opam-version: "2.0"
+maintainer: "Christophe Troestler <Christophe.Troestler@umons.ac.be>"
+authors: [ "Christophe Troestler <Christophe.Troestler@umons.ac.be>" ]
+license: "LGPL-3.0 with OCaml linking exception"
+homepage: "https://github.com/Chris00/L-BFGS-ocaml"
+dev-repo: "git+https://github.com/Chris00/L-BFGS-ocaml.git"
+bug-reports: "https://github.com/Chris00/L-BFGS-ocaml/issues"
+doc: "https://Chris00.github.io/L-BFGS-ocaml/doc"
+tags: [ "science" "numerics" "optimization" "minimization" "maximization"
+        "bound-constrained" "large-scale" ]
+build: [
+  ["dune" "subst"] {pinned}
```

Comment:
```suggestion
  ["dune" "subst"] {dev}
```

This causes a linter warning. I believe if you use Dune >= 2.7 to make the publish, it should automatically use `dev` instead of `pin` for this clause.
ref: https://github.com/ocaml/dune/blob/main/CHANGES.md#270-13082020

Reply:
Thanks.